.lando.yml

```diff
@@ -0,0 +1,45 @@
+name: drupal-orange-project
+recipe: drupal8
+config:
+  webroot: web
+  xdebug: true
+
+services:
```

Comment:
I _really_ recommend adding mailhog as a service. https://docs.devwithlando.io/services/mailhog.html

For example:
```
proxy:
  mailhog:
    - mail.drupal-orange-project.lndo.site
services:
  mailhog:
    hogfrom:
      - appserver
    portforward: true
```